FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,34 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+#include <Arduino.h>
+std_msgs::Int8 fuel_level_percent_full;
```

Comment:
By making this a float you're losing some of your available resolution. Good practice to take the data as close to raw as possible. Based on the ADC values you are describing below you could have a resolution of around 0.3%.

Reply:
Good point. I've gotten this fixed